<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sCleaning</class>
 <widget class="QWidget" name="sCleaning">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>876</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pbRun">
   <property name="geometry">
    <rect>
     <x>365</x>
     <y>740</y>
     <width>167</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: rgb(51, 204, 0);
border-width: 2px;
border-color: rgb(51, 204, 0);
border-style: solid;
border-radius: 10;
color: rgb(255, 255, 255);
font: 75 16pt &quot;Roboto Medium&quot;;
}
QPushButton:disabled { 
background-color: rgb(204, 244, 198);
border-width: 2px;
border-color: rgb(127, 143, 127);
border-style: solid;
border-radius: 10;
color: rgb(0, 0, 0);
font: 75 16pt &quot;Roboto Medium&quot;;
}
QPushButton:pressed {
background-color: rgb(24, 163, 3);
border-width: 2px;
border-color: rgb(24, 163, 3);
border-style: solid;
border-radius: 10;
color: rgb(255, 255, 255);
font: 75 16pt &quot;Roboto Medium&quot;;
}
QPushButton:down {
background-color: rgb(24, 163, 3);
border-width: 2px;
border-color: rgb(24, 163, 3);
border-style: solid;
border-radius: 10;
color: rgb(0, 0, 0);
font: 75 16pt &quot;Roboto Medium&quot;;
}
QPushButton:default {
background-color: rgb(241, 241, 241);
border-width: 2px;
border-color: rgb(21, 100, 192);
border-style: solid;
border-radius: 10;
color: rgb(0, 0, 0);
font: 75 16pt &quot;Roboto Medium&quot;;
}
</string>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbStop">
   <property name="geometry">
    <rect>
     <x>552</x>
     <y>740</y>
     <width>167</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: red;
border-width: 2px;
border-color: red;
border-style: solid;
border-radius: 10;
color: rgb(255, 255, 255);
font: 75 16pt &quot;Roboto Medium&quot;;
}
QPushButton:disabled { 
background-color: rgb(255, 174, 171);
border-width: 2px;
border-color: rgb(127, 143, 127);
border-style: solid;
border-radius: 10;
color: rgb(0, 0, 0);
font: 75 16pt &quot;Roboto Medium&quot;;
}
QPushButton:pressed {
background-color: rgb(194, 54, 11);
border-width: 2px;
border-color: rgb(194, 54, 11);
border-style: solid;
border-radius: 10;
color: rgb(255, 255, 255);
font: 75 16pt &quot;Roboto Medium&quot;;
}
QPushButton:down {
background-color: rgb(24, 163, 3);
border-width: 2px;
border-color: rgb(24, 163, 3);
border-style: solid;
border-radius: 10;
color: rgb(0, 0, 0);
font: 75 16pt &quot;Roboto Medium&quot;;
}
QPushButton:default {
background-color: rgb(241, 241, 241);
border-width: 2px;
border-color: rgb(21, 100, 192);
border-style: solid;
border-radius: 10;
color: rgb(0, 0, 0);
font: 75 16pt &quot;Roboto Medium&quot;;
}</string>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="twMeasuring">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>749</width>
     <height>711</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabWidget::pane { /* The tab widget frame */
    position: absolute;
	color: rgb(0, 0, 0);
	font: 75 16pt &quot;Roboto Medium&quot;;
	border: 0px solid #C4C4C3;
	border-bottom-color: #C2C7CB; /* same as the pane color */
   	border-bottom-left-radius: 30px;
   	border-bottom-right-radius: 30px;
    border-top-right-radius: 30px;
}

QTabWidget::tab-bar {
    alignment: left;
}

/* Style the tab using the tab sub-control. Note that
    it reads QTabBar _not_ QTabWidget */
QTabBar::tab {

    border: 1px solid #C4C4C3;
    border-bottom-color: #C2C7CB; /* same as the pane color */
    border-top-left-radius: 10px;
    border-top-right-radius: 10px;

    padding: 0px;
    height:0px;
	border-width: 0px;
	border-color: rgb(21, 100, 192);
	border-style: solid;

	font: 75 16pt &quot;Roboto Medium&quot;;
	color: rgb(0, 0, 0);
	

}

QTabBar::tab:selected, QTabBar::tab:hover {
	background-color: rgb(190, 190, 190);
}

QTabBar::tab:selected {
    border-color: #9B9B9B;
    border-bottom-color: #C2C7CB; /* same as pane color */
 border: 0px
}
</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>User Setup</string>
    </attribute>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>749</width>
       <height>851</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(241, 241, 241);
border-width: 0px;
border-color: rgb(21, 100, 192);
border-style: solid;
border-radius: 30;
color: rgb(0, 0, 0);
font: 75 16pt &quot;Roboto&quot;;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>29</x>
        <y>55</y>
        <width>683</width>
        <height>600</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
border-width: 1px;
border-color: rgb(21, 100, 192);
border-style: solid;
border-radius: 30;
color: rgb(0, 0, 0);
font: 75 16pt &quot;Roboto&quot;;

</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>59</y>
        <width>670</width>
        <height>595</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-width: 0px;
border-color: rgb(21, 100, 192);
border-style: solid;
border-radius: 30;
color: rgb(0, 0, 0);
font: 75 16pt &quot;Roboto&quot;;

</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="lblMessage">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>641</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Roboto Medium</family>
         <pointsize>24</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-width: 0px;
font: 75 24pt &quot;Roboto Medium&quot;;</string>
       </property>
       <property name="text">
        <string>Rinsing of cell</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QFrame" name="frLiveData">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>390</y>
         <width>531</width>
         <height>141</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame, QLabel {
   border-width:0px;
   border-radius: 0px;
   border-color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="lblTemperature">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>50</y>
          <width>211</width>
          <height>50</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(75, 225, 0);
font: 75 20pt &quot;Roboto Medium&quot;;</string>
        </property>
        <property name="text">
         <string>--.--</string>
        </property>
       </widget>
       <widget class="QLabel" name="lblPressure">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>50</y>
          <width>211</width>
          <height>50</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(75, 225, 0);
font: 75 20pt &quot;Roboto Medium&quot;;</string>
        </property>
        <property name="text">
         <string>---.--</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>5</y>
          <width>211</width>
          <height>50</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(77, 155, 232);
font: 75 20pt &quot;Roboto Medium&quot;;</string>
        </property>
        <property name="text">
         <string>Temperature</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>5</y>
          <width>211</width>
          <height>50</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(77, 155, 232);
font: 75 20pt &quot;Roboto Medium&quot;;</string>
        </property>
        <property name="text">
         <string>Pressure</string>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="lblCountDown">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>220</y>
         <width>651</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Roboto Medium</family>
         <pointsize>24</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-width: 0px;
font: 75 24pt &quot;Roboto Medium&quot;;</string>
       </property>
       <property name="text">
        <string>Current Cycle: -/-</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <zorder>lblMessage</zorder>
      <zorder>lblCountDown</zorder>
      <zorder>frLiveData</zorder>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pbHome">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>740</y>
     <width>167</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: red;
border-width: 2px;
border-color: red;
border-style: solid;
border-radius: 10;
color: rgb(255, 255, 255);
font: 75 16pt &quot;Roboto Medium&quot;;
}
QPushButton:disabled { 
background-color: rgb(255, 174, 171);
border-width: 2px;
border-color: rgb(127, 143, 127);
border-style: solid;
border-radius: 10;
color: rgb(0, 0, 0);
font: 75 16pt &quot;Roboto Medium&quot;;
}
QPushButton:pressed {
background-color: rgb(194, 54, 11);
border-width: 2px;
border-color: rgb(194, 54, 11);
border-style: solid;
border-radius: 10;
color: rgb(255, 255, 255);
font: 75 16pt &quot;Roboto Medium&quot;;
}
QPushButton:down {
background-color: rgb(24, 163, 3);
border-width: 2px;
border-color: rgb(24, 163, 3);
border-style: solid;
border-radius: 10;
color: rgb(0, 0, 0);
font: 75 16pt &quot;Roboto Medium&quot;;
}
QPushButton:default {
background-color: rgb(241, 241, 241);
border-width: 2px;
border-color: rgb(21, 100, 192);
border-style: solid;
border-radius: 10;
color: rgb(0, 0, 0);
font: 75 16pt &quot;Roboto Medium&quot;;
}</string>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
